app/src/behaviors/behavior_hold_tap.c

```diff
@@ -402,60 +404,72 @@ static inline const char *decision_moment_str(enum decision_moment decision_mome
 }
 
 static int press_hold_binding(struct active_hold_tap *hold_tap) {
+    struct zmk_behavior_binding binding = {.behavior_dev = hold_tap->config->hold_behavior_dev,
+                                           .param1 = hold_tap->param_hold};
     struct zmk_behavior_binding_event event = {
+        .binding = &binding,
         .position = hold_tap->position,
         .timestamp = hold_tap->timestamp,
+        .layer = hold_tap->layer,
```

Comment:
Thanks for adding these, I actually wanted to depend on them for a custom behavior but behaviors weren't dilige enough in passing through `layer`.